src/addressResolvers/cache.ts

```diff
@@ -0,0 +1,40 @@
+import { Address, Handle } from './utils';
+import redis from '../helpers/redis';
+
+const KEY = 'address-resolvers';
+const TTL = 24 * 60 * 60 * 1e3;
+
+export async function getCache(addresses: Address[]): Promise<Record<Address, Handle>> {
+  const transaction = redis.multi();
+  addresses.map(address => {
+    transaction.get(`${KEY}:${address}`);
+  });
+  const results = await transaction.exec();
+
+  return Object.fromEntries(
+    addresses
+      .map((address, index) => {
+        return [address, results[index]];
+      })
+      .filter(([, handle]) => handle !== null)
+  );
```

Comment:
```suggestion
  return Object.fromEntries(
    addresses
      .map((address, index) => [address, results[index]])
      .filter(([, handle]) => handle)
  );
```

Reply:
Suggestion is not working, since we only want to filter out `null` values, and keep empty string